src/lektorium/repo/local/server.py

```diff
@@ -157,6 +166,28 @@ def __init__(
         self.sessions_domain = os.environ.get('LEKTORIUM_SESSIONS_DOMAIN', None)
         self.lektorium_container = lektorium_container
 
+    @property
+    async def sessions(self):
+        def parse(containers):
+            for x in containers:
+                if 'lektorium.edit_url' not in x['Config']['Labels']:
+                    continue
+                session = {
+                    k[10:]: v
+                    for k, v in x['Config']['Labels'].items()
+                    if k.startswith('lektorium')
```

Comment:
'lektorium' could be moved to class constant and length of said constant could be used instead of "10"
Also will be used in `start` and `stop` then